### ISSUE TITLE
TK3.2 - Bus area micro journey

### ISSUE THREAD

**Issue by KaterinaKir** [wmcadigital/wmn-sprints#68]
**Epic:** Help users to understand which bus areas they're in

***
#### As a
commuter, leisure, business

#### I need to
understand which bus area I travel in

#### So that
I can buy a ticket that covers the right areas

***

#### What is a bus area?
- [x] User can understand an explanation of what bus areas are, written in plain English
- [x] User can see which bus zones exist

##### Map view
- [x] User can see all bus areas
- [x] User can select a particular bus area with clear boundaries on the map (Show Black Country n-bus zone, Coventry n-bus zone, Sandwell and Dudley n-bus zone, Regional n-bus zone, Walsall n-bus zone)
- [x] User can see a geographic map of bus stops and areas
- [x] User can focus the map by postcode
- [x] **??? - User can focus the map by bus stop name?**
- [x] User can choose two bus stops
- [x] User is told which bus areas they need to cover

##### List view
- [x] User can search for two bus stops by stop name
- [x] User can search for two bus stops by postcode
- [x] User is told which bus areas they need to cover

#### Technical
- [x] When a user has found areas, they are remembered on the ticket page if only one bus area is available. If multiple options are available then we do not pre-select.
***

**Comment by mrmjprice** [comment]
Will need integrating to #76
